Makefile

```diff
@@ -50,7 +50,7 @@ all: ocs-operator ocs-registry ocs-must-gather
 
 deps-update:
 	@echo "Running deps-update"
-	go mod tidy && go mod vendor
+	go mod tidy -compat=1.17 && go mod vendor
```

Comment:
Does OCS operator still have any dependency on golang 1.16? I couldn't get the prometheus code to compile without moving to 1.17.

Reply:
We recently moved to 1.17.